src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  switch (true) {
+    case (days < 3):
+      return 40 * days;
+
+    case (days >= 3 && days < 7):
+      return 40 * days - 20;
+
+    case (days >= 7):
+      return 40 * days - 50;
+  }
```

Comment:
```suggestion
  switch (true) {
    case (days < 3):
      return 40 * days;

    case (days >= 3 && days < 7):
      return 40 * days - 20;

    case (days >= 7):
      return 40 * days - 50;
  }
```

you should create variable for discount, adjust it's value in each case, and then return value from function, you should also create variable for daily rent price